src/scss/_packages.scss

```diff
@@ -2,80 +2,172 @@
+                        &:hover {
+                            @apply bg-black;
 
-                &.network {
-                    background-color: map-get($yellow, 200);
+                            .heading {
+                                @apply text-white;
+                            }
+
+                            svg {
+                                stroke: #FFFFFF;
+                            }
+                        }
+
+                        // Reset the background of the accordion item
+                        // when it is expanded.
+                        &.expanded {
+                            @apply bg-transparent;
+
+                            .heading {
+                                color: inherit;
+                            }
+
+                            svg {
+                                stroke: $svgDefaultStrokeColor;
+                            }
+                        }
+
+                        .heading, ul {
+                            @apply text-base;
+                        }
+
+                        ul {
+                            @apply list-none;
+                        }
+
+                        svg {
+                            stroke: $svgDefaultStrokeColor;
+                        }
+                    }
                 }
-            }
 
-            .native-package {
-                background-color: map-get($purple, 400);
-            }
+                .description {
+                    @apply overflow-ellipsis overflow-hidden;
 
-            .package-status {
-                &.ga, &.public-preview {
-                    background-color: map-get($fuchsia, 200);
+                    // The following is non-standard, but is weirdly supported
+                    // by all major browsers with the -webkit- prefix.
+                    // https://caniuse.com/?search=line-clamp
+                    display: -webkit-box;
+                    // Show ellipsis after 2 lines.
+                    -webkit-line-clamp: 2;
+                    -webkit-box-orient: vertical;
                 }
             }
         }
     }
 }
+
+.r-package-tags {
+    .package-tag {
+        @apply rounded;
+    }
+
+    .package-tag:not(last-child) {
+        @apply mr-4 p-2;
+    }
+
+    .package-category {
+        background-color: map-get($blue, 200);
+
+        &.cloud {
+            background-color: #BFCBAD;
+        }
+
+        &.database {
+            background-color: #E2F4AC;
+        }
+
+        &.infrastructure {
+            background-color: #E2F4AC;
+        }
+
+        &.monitoring {
+            background-color: #C4FEE5;
+        }
+
+        &.network {
+            background-color: #AAE8DF;
+        }
+
+        &.utility {
+            background-color: #FFC482;
+        }
+
+        &.version-control-system {
+            background-color: #FCE5AA;
+        }
+    }
+
+    .native-package {
+        background-color: map-get($purple, 400);
```

Comment:
Let's use the Tailwind helpers for these, yeah. (I'll stop commenting on these.)